tests/test_secure_random.py

```diff
@@ -2,20 +2,20 @@
 
 import tensorflow as tf
 import numpy as np
-from tf_encrypted.operations.secure_random import secure_random
+from tf_encrypted.operations.secure_random import seeded_random_uniform, random_uniform, seed as seed_gen
```

Comment:
hmm, why two test files?

Reply:
This one tests the wrapper around the secure random functions and the other one is more comprehensive. Could just have one here though.